### DISCUSSION TITLE
Cara Menampilkan Parent Title dari Sebuah Parent Gimana ya Kang?

### DISCUSSION THREAD

**Discussion by yuliakbard** [mecha-cms/mecha#225] [Q&A] [answered]
Kalau menampilkan Induk Judul dari sebuah halaman kan tinggal pake `$parent->title;` gimana cara menampilkan induk judul dari judul induk judul sebuah halaman kang?

**Comment by taufik-nurrohman** [comment, marked as answer]
Manual mas:

~~~ php
$folder = dirname($page->path);
if (
    is_file($parent = $folder . '.archive') ||
    is_file($parent = $folder . '.page')
) {
    $parent = new Page($parent);
} else {
    $parent = new Page;
}

echo $parent->title ?? i('Untitled');
~~~

Di versi 3.x.x nanti bakal ada [metode `parent()` bawaan](https://github.com/mecha-cms/x.page/commit/b1265f03e91fb40aaeb5c5e4363cf8d3f87cd295) untuk mempermudah:

~~~ php
echo $page->parent->title ?? i('Untitled');
~~~

**Reply by yuliakbard** [reply]
solved kang. makasih banyak kang. [lokersumatera.com](//lokersumatera.com) udah migrasi ke mecha, beberapa fitur yang ane mau, belum bisa direalisasikan, jadi bikin seadanya dulu. extension `x.import` gak bisa pake ya kang di mecha 2.6.4. khawatir ane Score Spam Website naik, karena bakal banyak error 404

**Reply by taufik-nurrohman** [reply]
Belum bisa mas. Statusnya masih eksperimen. Tapi kalau mau dicoba di Mecha versi lama dulu bisa. Sekedar untuk impor data aja. Nanti datanya dipindah ke Mecha versi baru.
